### PR TITLE
Add CCS desktops apps to default gnome favorites

### PR DESCRIPTION
**Pull request by glennmorris**

### CODE REVIEW

File: manifests/desktop.pp [hunk 1]
```diff
@@ -37,6 +41,12 @@
     }
   }
 
+  exec { 'dconf update':
```
Comment (jhoblitt) [inline manifests/desktop.pp:44]: The simp/dconf module is already in the control repo `Puppetfile`.  Perhaps this module could depend directly on that? https://forge.puppet.com/modules/simp/dconf/readme

File: manifests/desktop.pp [hunk 2]
```diff
@@ -37,6 +41,12 @@
...
+    command     => '/usr/bin/dconf update',
```
Comment (jhoblitt) [inline manifests/desktop.pp:45]: Exec commands should have acceptance test coverage as they historically have a high breakage rate.

**Review by jhoblitt — CHANGES_REQUESTED** [review]

File: manifests/desktop.pp (hunk 1 above)
Comment (glennmorris) [reply manifests/desktop.pp]: Alright, I pushed a new version that uses Somebody Else's Code to do the same thing.
As usual, the tests now fail in an opaque way.

File: manifests/desktop.pp (hunk 2 above)
Comment (glennmorris) [reply manifests/desktop.pp]: Punted to simp/dconf

File: manifests/desktop.pp (hunk 1 above)
Comment (glennmorris) [reply manifests/desktop.pp]: I managed to test the tests to pass.

**Review by jhoblitt — APPROVED** [review]
This looks good other than no test coverage.

**Pull request merged**